docs/apps/components/button.md

```diff
@@ -0,0 +1,83 @@
+# Button
+
+## Description
+
+A button can be used to trigger an event when it's clicked.
```

Comment:
A button can be used to perform an action through a number of events that can be triggered by interacting with it. e.g. performing an action when the click event is triggered.